test/fixtures/users.yml

```diff
@@ -156,9 +156,9 @@ user30:
   first_name: User
   last_name: "30"
 
-<% 21.times do |n| %>
+<% 22.times do |n| %>
 user_group_bot_account<%= (n) %>:
-  email: <%= "group_INXT_GRP_AAAAAAAAAA_bot_A3d#{n}Xc@localhost".downcase %>
+  email: <%= "group_INXT_GRP_AAAAAAAAAA_bot_A5d#{n}Xc@localhost".downcase %>
```

Comment:
**nitpick (blocking):** Can you update this to the new format that we talked about.

Reply:
Updated in d55982d7e963f7e8365a2f47f03f3b8060ec9230